### DISCUSSION TITLE
Hero Idea

### DISCUSSION THREAD

**Discussion by Sidders1943** [ValveSoftware/Dota2-Gameplay#9812] [Ideas]
Hello, I had an Idea for a new hero and I figured putting it here seemed as good a place as any.

So the general concept is a robot/construct which could be a metal golem or a mechanical construct depending on where you want to go with lore.

The main idea is that each of their basic abilities can be upgraded in one of two ways per game, this could either be upon maxing out the ability or upon levelling his ultimate, probably the latter.

I think it would either be Intelligence or Universal, though I would prefer the former.

I think I will lead with the ultimate, which would grant stacks when activated for a mana cost which are consumed by casting basic abilities. When consumed, they would make that cast of the ability have both upgrades, but also increase the cooldown and/or mana cost of that ability, the downside is reduced as the ultimate is levelled. I think the cooldown should be substantial and this ability should refresh the stacks upon use of refresher orb/shard to avoid complications. I don't think rubick should be able to steal this.

Q - Single target decaying slow similar to Venomous gale, but also affecting attack speed. 
Upgrade 1 - Turn it into a stun for part of the duration similar to wraithfire blast/slithereen crush.
Upgrade 2 -  Increase the duration of the slow and make it decay slower, possibly increase the strength of the attack speed slow.

W - Single target magical nuke.
Upgrade 1 - Change damage type to pierce BKB and increase cast range (Not sure if pure makes more sense from a balance perspective so could be physical if necessary).
Upgrade 2 - Make it AOE for wave-clear/farming.
Note - When both upgrades are used it would behave like frost blast rather than something like Zeus' aoe lightning bolt talent.

E - Single target shield buff similar to aphotic shield but without the dispell - maybe make it similar to empower in that it gives larger buffs when self cast. It could be physical or magic if required for balnce reasons.
Upgrade 1 - Provide move speed and attack speed to the target - possibly increase duration.
Upgrade 2 - Makes it apply a weak dispell on cast - possibly increase the shield value.

Aghanim's Shard - Make Q and E undispellable by weak dispells.

Aghanim's Sceptre - Each cast of an ability that consumes stacks from the ultimate also reduces the cooldowns of other basic abilities. Possibly increase stacks granted by ultimate. This avoids rubick destroying you since he cannot use the ult effectively.

General hero properties - Low starting stats, but good stat gain upon levelling the hero is being improved/upgraded throughout the match so it makes sense. Poor base hp regen, but decent base mana regen.

I think the hero's spells should be mana expensive, but very impactful later in the game, so they should be poor with one point in them but be very powerful when maxed out.

Either bad base damage or low attack range to avoid it being a good harrassing support in the lane outside of its' spells. I think it should be played as position 4 or position 2.

Morph and Rubick should use the upgrade that you have selected for the abilities.

Thanks for reading it if you got this far.

**Comment by johnsantos2** [comment]
¿Crees que deberías descargar un mod gratuito en Tech Loky?
